app/game/game.js

```diff
@@ -0,0 +1,55 @@
+const { ROLES } = require("../constants");
+const GAMES = {
+    
+};
+
+const GAME = (humanUsername, aiUsername) => {
+    let gameLength = 4*60*1000;
+    let HUMAN_USERNAME = humanUsername;
+    let AI_USERNAME = aiUsername;
+    let room = "0-0";
+
+    const getRole = (username) => {
+        if (username == HUMAN_USERNAME) {
+            return ROLES.HUMAN;
+        }
+        if (username == AI_USERNAME) {
+            return ROLES.AI;
+        }
+        return null;
+    }
```

Comment:
I'm noticing that many of these changed files rely on storing and checking username strings. I think nearly all of these occurrences should be replaced by clientIDs / tokens once we get that running. Usernames exist only to be human-readable identifiers of other players to the client. The server shouldn't care what your username is, it only cares who you are. Guests (non-logged-in users) may not have usernames at all, so we need a better way to identify them.

Reply:
We can assign each person a clientId (which could be used as a "username" for non-logged in guests), currently using username since it's intuitive and I agree clientId would be great once it's up, for now this should be easy way to follow logic